### PR TITLE
Avoid confusion with Feature Toggles link

### PR DESCRIPTION
**Pull request by saraycp**
Fixes: #14417

For admins, the Feature Toggle link is moved to a new tab inside Configuration.

Screenshot 2024-04-15 at 19-38-12 Open Build Service

Staff users need to use the link but don't have access to Configuration. Therefore, the link stays in the same place for them but uses a clearer text "_Global Feature Toggles_" and opens a confirmation message.

Screenshot from 2024-04-15 19-35-10

In the future, we could have a section similar to Configuration but only for Staff, something like "Staff Global Settings" to include all the global actions they can do. Right now, it's not worth doing so, just to include the link to Flipper dashboard.

### CODE REVIEW

File: src/api/app/views/webui/global_feature_toggles/index.html.haml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+- @pagetitle = 'Global Feature Toggles'
```
[the remark this answers is not in the document]
Comment (saraycp) [reply src/api/app/views/webui/global_feature_toggles/index.html.haml:1]: I think it doesn't harm. The more obvious the better...

**Comment by hennevogel** [comment]
IMHO renaming the link is enough, what value is there to send admins to a page with a link?

**Comment by saraycp** [comment]
> IMHO renaming the link is enough, what value is there to send admins to a page with a link?

Having all the admin-only tasks (or global tasks) in the same place is coherent. Having most of them under Configuration but one of them under the Places menu is more confusing.

**Comment by saraycp** [comment]
@hellcp-work @hennevogel I'd use an approval here :pray:

**Pull request merged**